app/src/main/java/org/p2p/wallet/bridge/send/BridgeSendInteractor.kt

```diff
@@ -49,10 +48,10 @@ class BridgeSendInteractor(
         val sendTransaction = repository.transferFromSolana(
             userWallet = userWallet,
             feePayer = feePayer,
-            source = source,
+            source = SolAddress(token.publicKey),
             recipient = recipient,
             mint = SolAddress(tokenMint),
-            amount = amount.toPlainString()
+            amount = amountInLamports.toString()
```

Comment:
amount  -> amountInLamports

type String doesn't give us any information what's inside and what format it has